vars/perfRegressionParallelPipeline.groovy

```diff
@@ -154,6 +154,10 @@ def call(Map pipelineParams) {
             string(defaultValue: 'next',
                    description: 'Default branch to be used for scylla and other repositories. Default is "next".',
                    name: 'byo_default_branch')
+
+            booleanParam(defaultValue: false,
```

Comment:
@juliayakovlev @roydahan  thinking  if we want this on by default or off by default ?